apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -26,10 +28,43 @@ export const resourceRouter = router({
 
       return resource
     }),
+  getFolderChildrenOf: protectedProcedure
+    .input(getChildrenSchema)
+    .query(async ({ input: { resourceId, cursor: offset, limit } }) => {
+      let query = db
+        .selectFrom("Resource")
+        .select(["title", "permalink", "type", "id"])
+        .where("Resource.type", "in", ["RootPage", "Folder"])
+        .$narrowType<{
+          type: "Folder" | "RootPage"
+        }>()
```

Comment:
do you think we shuold also allow `Collection` here? i saw this is only used for `move`, which ew disallow for collections atm but just food for thought

Reply:
ya, collections not here solely because you cannot move into collections.